src/src/AdminBundle/Command/AddMockedDataCommand.php

```diff
@@ -0,0 +1,185 @@
+    }
+
+    private function getRandomEvent($em)
+    {
+        return $this->getRandomObject($em, Event::class);
+    }
+
+    private function getRandomEventType($em)
+    {
+        return $this->getRandomObject($em, EventType::class);
+    }
+
+    private function makeEmail($first, $last)
+    {
+        return strtolower($first.'@'.$last.'.com');
+    }
+
+    private function randomEmail()
+    {
+        return $this->makeEmail($this->randomFirstName(), $this->randomLastName());
+    }
+
+    protected function configure()
+    {
+        $this->setName('nobelisti:add:mocked-data')
+            ->setDescription('Adds mocked data for development purposes');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $em = $this->getContainer()->get('doctrine')->getManager();
+
+//        $this->createAdmins($em);
+//        $this->createLanguages($em);
+//        $this->createOrganizers($em);
+//        $this->createAttendees($em);
+//        $this->createEventTypes($em);
+//        $this->createEvents($em);
+
+        // todo: $this->createRegistrations($em);
+        // todo: $this->assignOrganizersToEvents($em);
+        // todo: $this->assignLanguagesToEvents($em);
+        // todo: $this->assignLanguagesToEventTypes($em);
+        // todo: $this->createRegistrations($em);
+
+        $em->flush();
+    }
+
+    private function createAdmins($em)
+    {
+        for ($i = 0; $i < 10; ++$i) {
+            $admin = new Admin();
+            $admin->setEmail($this->randomEmail());
+            $admin->setPassword('');
+            $em->persist($admin);
+        }
+    }
+
+    private function createLanguages($em)
+    {
+        $lang1 = new Language();
+        $lang1->setCode('en_US');
+        $lang1->setLanguage('English');
+
+        $lang2 = new Language();
+        $lang2->setCode('sk_SK');
+        $lang2->setLanguage('Slovak');
+
+        $em->persist($lang1);
+        $em->persist($lang2);
+    }
+
+    private function createOrganizers($em)
+    {
+        for ($i = 0; $i < 10; ++$i) {
+            $organizer = new Organizer();
+            $organizer->setEmail($this->randomEmail());
+            $em->persist($organizer);
+        }
+    }
+
+    private function createAttendees($em)
+    {
+        for ($i = 0; $i < 100; ++$i) {
+            $attendee = new Attendee();
+            $attendee->setFirstName($this->randomFirstName());
+            $attendee->setLastName($this->randomLastName());
+            $attendee->setLanguages($this->getRandomLanguage($em));
+            $attendee->setUnsubscribed((bool) random_int(0, 1));
+            $attendee->setEmail($this->makeEmail($attendee->getFirstName(), $attendee->getLastName()));
+            $em->persist($attendee);
+        }
+    }
+
+    private function createEventTypes($em)
+    {
+        for ($i = 0; $i < 5; ++$i) {
+            $type = new EventType();
+            $type->setTemplate('default');
```

Comment:
daj pozor, default template z nejakeho dovodu nefunguje, teda ze nieco tam chyba a je bugnuty

Reply:
tak ho bude treba opravit, mozes si to vziat na starost :)